### PR TITLE
Integrate internal feedback

### PR DESCRIPTION
**Pull request by bwdGitHub**
I've modified the example following internal feedback. This version should more closely resemble other documentation examples in style and content.

Some notable changes:

1. The targets are no longer normalized. This means the training losses are on a different scale to before. This seemed to be OK - the final results still look good. Inputs are normalized via the `image3dInputLayer` defaults.
2. Use of [`combinations`](https://uk.mathworks.com/help/matlab/ref/combinations.html) over doing this manually to create all combinations of the varying parameters.
3. Use of [`fillmissing`](https://uk.mathworks.com/help/matlab/ref/fillmissing.html) over manually finding `NaN` and replacing it.
4. Only solve the PDE for training data at the final time. This drastically reduces the size of the results objects since you're only storing the solution at the final time - from about 2GB to 400MB.

There's a couple extra changes I'll try push up on this branch:

1. Use [`bounds`](https://uk.mathworks.com/help/matlab/ref/bounds.html) over finding `min` and `max` independently.
2. Swap from `meshgrid` to `ndgrid` This saves some awkward `permute`-s in the `griddedInterpolant` call used at the test/inference section at the end.

### CODE REVIEW

**Comment by bwdGitHub** [comment]
I've addressed the extra changes 1. and 2. in the latest commit:
1. Use `bounds` - simplifies the code over separate `min` and `max` calls.
2. Use `ndgrid` rather than `meshgrid`, integrates better with `griddedInterpolant` later on.
